packages/libncurses/ncurses-kbs.patch

```diff
@@ -1,25 +1,16 @@
-diff -up ncurses-6.4-20230107/misc/terminfo.src.kbs ncurses-6.4-20230107/misc/terminfo.src
---- ncurses-6.4-20230107/misc/terminfo.src.kbs	2023-01-09 14:47:05.097093771 +0100
-+++ ncurses-6.4-20230107/misc/terminfo.src	2023-01-09 14:47:05.100093766 +0100
-@@ -6711,7 +6711,7 @@ rxvt-basic|rxvt terminal base (X Window
- 	enacs=\E(B\E)0, flash=\E[?5h$<100/>\E[?5l, home=\E[H,
- 	ht=^I, hts=\EH, ich=\E[%p1%d@, il=\E[%p1%dL, il1=\E[L,
- 	ind=\n, is1=\E[?47l\E=\E[?1l,
--	is2=\E[r\E[m\E[2J\E[H\E[?7h\E[?1;3;4;6l\E[4l, kbs=^H,
-+	is2=\E[r\E[m\E[2J\E[H\E[?7h\E[?1;3;4;6l\E[4l,
- 	kcbt=\E[Z, kmous=\E[M, rc=\E8, rev=\E[7m, ri=\EM, rmacs=^O,
- 	rmir=\E[4l, rmkx=\E>, rmso=\E[27m, rmul=\E[24m,
- 	rs1=\E>\E[1;3;4;5;6l\E[?7h\E[m\E[r\E[2J\E[H,
-@@ -6722,7 +6722,7 @@ rxvt-basic|rxvt terminal base (X Window
- 	    %p9%t\016%e\017%;,
- 	sgr0=\E[0m\017, smacs=^N, smir=\E[4h, smkx=\E=, smso=\E[7m,
- 	smul=\E[4m, tbc=\E[3g, use=xterm+alt47, use=vt100+enq,
+diff -up ncurses-6.5-20240504/misc/terminfo.src.kbs ncurses-6.5-20240504/misc/terminfo.src
+--- ncurses-6.5-20240504/misc/terminfo.src.kbs	2024-05-08 17:33:47.305060391 +0000
++++ ncurses-6.5-20240504/misc/terminfo.src	2024-05-08 17:33:37.439196377 +0000
+@@ -6707,7 +6707,7 @@ rxvt-basic|rxvt terminal base (X Window
+ 	sgr0=\E[0m\017, smir=\E[4h, smkx=\E=, smso=\E[7m,
+ 	smul=\E[4m, use=ansi+csr, use=ansi+idl, use=ansi+local,
```

Comment:
This patch originated in Fedora, and we can probably just drop it as Fedora has done:
https://src.fedoraproject.org/rpms/ncurses/c/69028138750e85c719e16f04fcdaaf8c9f1509f7?branch=main